tests/fixtures/file_fixtures.py

```diff
@@ -29,8 +29,8 @@
 
 from .keypair_fixtures import KeypairFixture, generate_keypair_fixture  # noqa: F401
 
-BUCKET_ID = "test"
-OBJECT_ID = "random-data"
+BUCKET_ID = "test-bucket"
+OBJECT_ID = "test-object"
 FILE_SIZE = 50 * 1024**2
 PART_SIZE = 16 * 1024**2
 
```

Comment:
The `sys.set_int_max_str_digits(256 * 1024**2)  # type: ignore` in next line is causing the test plugin of vscode to fail.

Reply:
Does not work with normal mode, but they run in debug mode and normally in the CLI...
Not sure what's going on here

Reply:
Moved into fixture.
If this does not fix it, this is more of  a vscode issue